commit 1e3cd48ab49be1eaa689064d1503f74c75ad10b1
Author: Bjorn Andersson <bjorn.andersson@linaro.org>
Date:   Fri Jan 27 02:06:36 2017 -0800

    remoteproc: qcom: mdt_loader: Don't overwrite firmware object
    
    commit 3e8b571a9a0881ba3381ca0915995696da145ab8 upstream.
    
    The "fw" firmware object is passed from the remoteproc core and should
    not be overwritten, as that results in leaked buffers and a double free
    of the the last firmware object.
    
    Fixes: 051fb70fd4ea ("remoteproc: qcom: Driver for the self-authenticating Hexagon v5")
    Signed-off-by: Bjorn Andersson <bjorn.andersson@linaro.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1327_1e3cd48ab49be1eaa689064d1503f74c75ad10b1.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1327_1e3cd48ab49be1eaa689064d1503f74c75ad10b1.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
